src/Core/Models/Data/CipherAttachment.cs

```diff
@@ -32,6 +33,13 @@ public string SizeString
 
             public string FileName { get; set; }
             public string Key { get; set; }
+
+            [DefaultValue("attachments")]
```

Comment:
I would remove this, and the reference to `System.ComponentModel` and simplify this with a constructor setter (see below)